### ISSUE TITLE
not compatible with angular 5 after version beta 4

### ISSUE THREAD

**Issue by AlexKhymenko** [500tech/angular-tree-component#454]
Not compatible after version 5.beta-5 

https://github.com/angular/angular/blob/master/CHANGELOG.md#500-beta5-2017-08-29

common: remove deprecated NgTemplateOutlet#ngOutletContext

common: NgTemplateOutlet#ngOutletContext has been removed as it was deprecated since v4. Use NgTemplateOutlet#ngTemplateOutletContext instead.

error message
```
   To allow any property add 'NO_ERRORS_SCHEMA' to the '@NgModule.schemas' of this component. ("
      <ng-container
        [ngTemplateOutlet]="templates.treeNodeFullTemplate"
        [ERROR ->][ngOutletContext]="{ $implicit: node, node: node, index: index, templates: templates }">
      </ng-c"): ng:///TreeModule/TreeNodeComponent.html@22:8
```

**Comment by adamkleingit** [comment]
Will wait for Angular 5 to be released and then make it compatible

**Comment by AlexKhymenko** [comment]
Thank You they are promising official release next week.

**Comment by ghaiat** [comment]
Hello, any update? Angular 5 has been released ...
Cheers

**Comment by jycouet** [comment]
Hey, same thing here :) v5.0 it out :)

**Comment by jycouet** [comment]
I guess that https://github.com/500tech/angular-tree-component/pull/460 is doing the job.
Could you have a look please? 
Many thanks

**Comment by colordove** [comment]
Hey, I have the same problem, hope to have a new version. thanks!

**Comment by matt-senseye** [comment]
Seeing the same issue here when trying to update to Angular 5.0.0

**Comment by osnoser1** [comment]
Same problem!

**Comment by burakkilic** [comment]
Same problem here

**Comment by cipchk** [comment]
hope this issue can be sloved in time.

**Comment by johaven** [comment]
+1

**Comment by burakkilic** [comment]
It's solved in the master branch, but not available for npm I guess?

**Comment by jycouet** [comment]
Yes I guess so @burakkilic.
Do you have rights to push it to npm @adamkleingit?

Thx

**Comment by pumano** [comment]
ng-container and ngOutletContext is deprecated and/or removed in Angular 5 (for details check angular 5 changelog). 

angular-tree-component should be angular 5 compliant for using it with angular 5.

**Comment by adamkleingit** [comment]
Merged and will be published soon


*Adam Klein*
*Chief Technology Officer*



*Tel.*   +972-52-7475633 <+972527475633>
*Web*  500tech.com <http://www.500tech.com/> | github.com/500tech |
@adamklein500 <http://twitter.com/adamklein500>

**Comment by finalxcode** [comment]
cheer up!

**Comment by nadhir-falta** [comment]
Has this one got published yet ?

**Comment by isaacplmann** [comment]
@adamkleingit is at Angular Connect this week, so my guess is that publishing this might take a while.  Enjoy the conference, Adam!  We'll survive a week without upgrading to Angular 5.

**Comment by SunYuanOnly** [comment]
+1

**Comment by matt-senseye** [comment]
@adamkleingit are you able to provide an update on when this is likely to get published? Thanks

**Comment by adamkleingit** [comment]
Hi,
published in 5.2.0. Thanks for the patience.
@isaacplmann Thanks! Did you enjoy my talk? :)

**Comment by finalxcode** [comment]
@adamkleingit  thanks
